ZipBuilder/Sources/ZipBuilder/main.swift

```diff
@@ -16,19 +16,17 @@
 
 import Foundation
 
+// Delete the cache directory, if it exists.
+do {
+  let cacheDir = try FileManager.default.firebaseCacheDirectory()
```

Comment:
Looks like cache was overloaded before - and now just means one thing?

Reply:
Yes, was overloaded before. the cache directory here is more of a temporary scratch folder for us to write to and is not used between builds. Perhaps there's a better name for it?